unlock-app/src/pages/dashboard.js

```diff
@@ -9,7 +9,7 @@ import CreatorLocks from '../components/creator/CreatorLocks'
 import withConfig from '../utils/withConfig'
 import { pageTitle } from '../constants'
 
-export const Dashboard = ({account, network, locks}) => {
+export const Dashboard = ({ account, network, locks, conversion }) => {
```

Comment:
Please cleanup `conversion` from this file too ;)

Reply:
done